tests/phpunit/tests/multisite/isUploadSpaceAvailable.php

```diff
@@ -74,7 +74,7 @@ public function test_is_upload_space_available_upload_space_0_defaults_to_100()
 			$available = is_upload_space_available();
 			remove_filter( 'pre_get_space_used', array( $this, 'filter_space_used_small' ) );
 
-			$this->assertFalse( $available );
+			$this->assertTrue( $available );
```

Comment:
This appears to constitute a BC break.

See [r35016](https://core.trac.wordpress.org/changeset/35016), which allowed a `blog_upload_space` value of `0` to disable uploads. The changeset also changed the assertion in this test to `assertFalse()` to protect the behaviour.